### PR TITLE
[Dataminr Pulse] Release 106

### PR DESCRIPTION
**Pull request by crestdatasystems**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
- Updated the layout to show the important incident fields and rearranged the layout sections.
- Updated IOC extraction to now extract from Cyber terms instead of the entire raw incident data.
- Updated the Dataminr Pulse Alert incident type to prevent the default playbook from autorunning
 
## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Review by MichaelYochpaz — APPROVED** [review]
Hey, thank you for the contribution.
Looks good from my side of the review, just waiting for @melamedbn's approval.

**Review by melamedbn — APPROVED** [review]

**Pull request merged**
